source/style-guide/style/text-formatting.txt

```diff
@@ -4,603 +4,1636 @@
+
+            On the :guilabel:`Server Settings` page, click
+            :guilabel:`Check Name`, type your password, and then click
+            :guilabel:`OK`.
+
+   Panes, named and unnamed
+     Format using ``:guilabel:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               To verify that your TLS binding works, select your
+               website in the :guilabel:`Connections` pane (if it's not
+               already selected) and then click :guilabel:`Browse
+               ipAddress (https)` in the :guilabel:`Actions` pane.
+
+               In the navigation pane, select
+               :guilabel:`Composing Email`.
+
+          - To verify that your TLS binding works, select your website
+            in the :guilabel:`Connections` pane (if it's not already
+            selected) and then click :guilabel:`Browse ipAddress
+            (https)` in the :guilabel:`Actions` pane.
+
+            In the navigation pane, select :guilabel:`Composing Email`.
+
+   Parameter names
+     Format using ``:option:`` role.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+               :linenos:
+
+               The size of the oplog is only configurable during the
+               first run using the ``--oplogSize`` argument to the
+               :binary:`~bin.mongod` command, or preferably, the
+               :setting:`~replication.oplogSizeMB` setting in the
+               MongoDB configuration file. If you do not specify this
+               on the command line before running with the
+               :option:`mongod --replSet` option, :binary:`~bin.mongod`
+               creates a default sized oplog.
+
+          - The size of the oplog is only configurable during the first
+            run using the ``--oplogSize`` argument to the
+            :binary:`~bin.mongod` command, or preferably, the
+            :setting:`~replication.oplogSizeMB` setting in the MongoDB
+            configuration file. If you do not specify this on the
+            command line before running with the ``--replSet`` option,
+            :binary:`~bin.mongod` creates a default sized oplog.
+
+   Paths
+     Format using ``monospace`` text.
+
+     .. list-table::
+        :widths: 50 50
+        :header-rows: 1
+
+        * - Code Example
+          - Rendered Example
+
+        * - .. code-block:: rst
+
+               The path to Perl is ``#!/usr/bin/perl -w``.
+
+               In the URI path
+               ``https://cloud.mongodb.com/api/public/v1.0``, the API
+               version is 1.
+
+          - The path to Perl is ``#!/usr/bin/perl -w``.
+
+            In the URI path
+            ``https://cloud.mongodb.com/api/public/v1.0``,
+            the API version is 1.
+
+   Permissions
```

Comment:
I would omit this since it's not clear what type of permissions you're referring to. Based on the rst snippet, I think you're just talking about the word "permissions", but without describing it as such, someone might think of unix permissions (rwx) or other authorization roles.